webkit_server.py

```diff
@@ -513,12 +520,47 @@ def issue_command(self, cmd, *args):
     self._writeline(cmd)
     self._writeline(str(len(args)))
     for arg in args:
-      arg = str(arg)
-      self._writeline(str(len(arg)))
-      self._sock.sendall(arg.encode("utf-8"))
+        self.send_arg(arg)
```

Comment:
This file seems to be using an indentation of 2 spaces rather than 4 consistently throughout the file. I suggest correcting the indentation of line 523.